spec/vcloud/core/vapp_spec.rb

```diff
@@ -154,6 +154,42 @@
       end
     end
 
+  context "power off" do
+    context "successful power off" do
+      before(:each) do
+        @fog_vapp_body = {
+          :name => "Webserver vapp-1",
+          :href => "https://api.vcloud-director.example.com/api/vApp/vapp-63d3be58-2d5c-477d-8410-267e7c3c4a02",
+          :Link => [{
+          :rel => "stopped",
```

Comment:
The indentation here looks wrong to me; I think the symbols nested under Link should be indented two points.